messaging/src/main/java/org/axonframework/messaging/unitofwork/MessageProcessingContext.java

```diff
@@ -112,7 +112,7 @@ public void addHandler(Phase phase, Consumer<UnitOfWork<T>> handler) {
     public void setExecutionResult(ExecutionResult executionResult) {
         Assert.state(this.executionResult == null || executionResult.isExceptionResult(),
                      () -> String.format("Cannot change execution result [%s] to [%s] for message [%s].",
-                        message, this.executionResult, executionResult));
+                        executionResult , this.executionResult, message));
```

Comment:
Thanks for pointing this out. I assume however that the parameter order should be `this.executionResult`, `executionResult`, and then the `message`.
```suggestion
                        this.executionResult , executionResult, message));
```